### PR TITLE
box: document limitations; fixes for WebRTC

### PR DESCRIPTION
**Pull request by iameli**
-   Changed the hardcoded streams in the database snapshots to have
    easy-to-remember stream keys like `2222-2222-2222-2222`
-   Changed the built-in streams to use the H264ConstrainedHigh profile so there
    are no B-Frames in the output
-   Moved all references from `127.0.0.1` to `localhost`; this is needed for
    WebRTC/Coturn to work properly
-   Removed outdated references to `GOOS=linux` and `KILL=true`; these are the
    defaults now

### CODE REVIEW

**Comment by iameli-streams** [comment]
-   Changed the hardcoded streams in the database snapshots to have
    easy-to-remember stream keys like `2222-2222-2222-2222`
-   Changed the built-in streams to use the H264ConstrainedHigh profile so there
    are no B-Frames in the output
-   Moved all references from `127.0.0.1` to `localhost`; this is needed for
    WebRTC/Coturn to work properly
-   Removed outdated references to `GOOS=linux` and `KILL=true`; these are the
    defaults now

File: Makefile [hunk 1]
```diff
@@ -219,6 +219,7 @@ box-dev: scripts
 	-v $$(realpath config):/etc/livepeer:ro \
 	-v $$(realpath ./coredumps):$$(realpath ./coredumps) \
 	-e CORE_DUMP_DIR=$$(realpath ./coredumps) \
+	$(shell for line in $$(cat .env 2>/dev/null || echo ''); do printf -- "-e $$line "; done) \
```
Comment (pwilczynskiclearcode) [inline Makefile:222]: Can we use `docker run --env-file=...` https://docs.docker.com/engine/reference/commandline/run/#env ?

File: docs/livepeer-in-a-box.md [hunk 2]
```diff
@@ -1,10 +1,95 @@
...
+## Running the Box
+
+First, select a directory for persisting your database and video content; in
+this example we will be using `$HOME/livepeer-in-a-box`.
 
-## Dependencies
+```shell
+BOX_DIR="$HOME/livepeer-in-a-box"
+mkdir -p $BOX_DIR
+docker run \
```
Comment (pwilczynskiclearcode) [inline docs/livepeer-in-a-box.md:39]: Should we encourage to use one of `make` commands?

**Review by thomshutt — APPROVED** [review]

File: docs/livepeer-in-a-box.md (hunk 2 above)
Comment (iameli) [reply docs/livepeer-in-a-box.md:39]: No, 'cause these are directions for someone that might be running it without developing it

**Pull request merged**